Updated owners for embedding related code

zhux97 opened the pull request.

zhux97 committed:
Add macros for converting embedding MuDst to PicoDst

plexoos committed:
Merge branch 'star-bnl:main' into 'zhux97:main'

zhux97 committed:
Merge branch 'star-bnl:main' into main

zhux97 committed:
Tuned t0 offset according to FXT26p5 data

by Yue-Hang Leung, Yuri Fisyak

zhux97 committed:
Delete TpcResponseSimulator.20180605.071409.C

zhux97 committed:
Merge branch 'star-bnl:main' into main

zhux97 committed:
Update CODEOWNERS

plexoos commented:
Argh... I must have hit rebase instead of squash and now we have two unintended commits from this PR on the main branch 5dfd787943e71ea54057d9b6838d0e518f39d619 and 7530a7b74cf14dcb993187b1d512e41d37237e9b

@zhux97 Can you explain what those commits are and why you put them on this branch?

zhux97 commented:
Hi Dmitri,

This was the operation on my forked code, added the TpcRS tuning for FXT26p5 data and removed it later since Yuri has a same pull request now.
I still need to learn how to remove those unintended commits in the pull request, sorry about it.




> On Aug 12, 2021, at 5:27 AM, Dmitri Smirnov ***@***.***> wrote:
> 
> 
> Argh... I must have hit rebase instead of squash and now we have two unintended commits from this PR on the main branch 5dfd787 and 7530a7b
> 
> @zhux97 Can you explain what those commits are and why you put them on this branch?
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or unsubscribe.
> Triage notifications on the go with GitHub Mobile for iOS or Android.
>
